src/Voron/Platform/Pal.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+
+// ReSharper disable SwitchStatementMissingSomeCases
+// ReSharper disable InconsistentNaming
+
+namespace Voron.Platform
+{
+    public static unsafe class Pal
+    {
+        static Pal()
+        {
+            var toFilename = LIBRVNPAL;
+            var fromFilename = $"{toFilename}.win.x64.dll";
```

Comment:
why is this initialized here?

Reply:
it works (because executed before lib load) and makes logic to put it here in terms of context

Reply:
Why is it init to `$"{toFilename}.win.x64.dll";` ?